### PR TITLE
Check if original partitions are mounted too

### PR DESCRIPTION
**Pull request by VladimirSlavik**
Resolves: rhbz#1751698

### CODE REVIEW

**Comment by VladimirSlavik** [comment]
This could be also moved to master (34) only, if it's the wrong time to break things...

File: pyanaconda/modules/storage/checker/utils.py [hunk 1]
```diff
@@ -429,15 +431,30 @@ def verify_mounted_partitions(storage, constraints, report_error, report_warning
         if not disk.partitioned:
             continue
 
-        for part in disk.format.partitions:
-            part_dev = storage.devicetree.get_device_by_path(part.path)
+        already_checked_partitions = set()
```
Comment (poncovka) [inline pyanaconda/modules/storage/checker/utils.py:434]: It might be easier to first collect a set of partitions that need to be checked and then check these partitions in another loop. This solution is a little hard to follow.

Comment (VladimirSlavik) [reply pyanaconda/modules/storage/checker/utils.py]: Done...

**Review by poncovka — APPROVED** [review]
Looks good to me. Thanks!

**Review by jkonecny12 — APPROVED** [review]
Looks good to me.
